Dockerfile

```diff
@@ -28,17 +25,14 @@ WORKDIR /app/${package}
 
 # Run tests and return coverage analysis
 RUN python -m coverage run tests/test_${package}.py \
- && python -m coverage html -d /htmlcov --omit=/usr/local/*
+&& python -m coverage html -d /htmlcov --omit=/usr/local/*
```

Comment:
I dunno 'bout you, but I like to have continuation lines indented (although, I would indent four or eight spaces rather than _one_, but...).

Reply:
The single indent got accidentally dropped. The reason for using a single space is that is aligns the commands neatly. I could otherwise do: